converters/fromZigbee.js

```diff
@@ -957,7 +957,7 @@ const converters = {
     },
     ias_smoke_alarm_1: {
         cluster: 'ssIasZone',
-        type: 'commandStatusChangeNotification',
+        type: ['commandStatusChangeNotification', 'attributeReport', 'readResponse'],
         convert: (model, msg, publish, options, meta) => {
             const zoneStatus = msg.data.zonestatus;
```

Comment:
For the `attributeReport`/`readResponse` the attribute is `zoneStatus`, not `zonestatus`. This line should become:

```js
const zoneStatus = msg.type === 'commandStatusChangeNotification' ? msg.data.zonestatus : msg.data.zoneStatus;
```

Reply:
Oh, good catch. Sorry for that mistake.
(Perhaps it would be wise to name them systematically to avoid such confusion in the future)